src/validate.py

```diff
@@ -299,6 +302,11 @@ def loop(validation_args_file: str, task_id: str = None):
     last_successful_request_time = [time.time()] * len(task_id_list)
 
     while True:
+        for root, dirs, _ in os.walk(file_utils.default_cache_path):
```

Comment:
Will the `default_cache_path` still be accurate if someone has set `HF_HOME` to a different path?

Reply:
yes, the code about [file_utils.default_cache_path](https://huggingface.co/transformers/v4.1.1/_modules/transformers/file_utils.html)
```
hf_cache_home = os.path.expanduser(
    os.getenv("HF_HOME", os.path.join(os.getenv("XDG_CACHE_HOME", "~/.cache"), "huggingface"))
)
default_cache_path = os.path.join(hf_cache_home, "transformers")
```